cli/src/main/kotlin/OrtMain.kt

```diff
@@ -202,5 +203,6 @@ fun fixupUserHomeProperty() {
  */
 fun main(args: Array<String>) {
```

Comment:
Also, wouldn't the more correct fix anyway be to change the return type of `main` to `Int`?

Reply:
It might be, but I'd see that out of scope of the commit. Do you agree?

I think it is not more correct as `exitProcess` actuall returns `Nothing`, see also its signature.

Reply:
> It might be, but I'd see that out of scope of the commit. Do you agree?

I don't think it's out of scope, but I was mislead anyway, as clikt's `main()` does not return `Int` (but `Unit`), so it'd be wrong for us to return `Int` here.